README.md

```
@@ -1,27 +1,55 @@
 Tool to automate the creation of Release Notes for Podman Desktop
 
 ## Usage
+
 - Generate release notes `pnpm generate`
 - Test `pnpm test`
 
 ### Examples
 
 1. Generate release notes using AI Lab service
+
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)